cddl/contrib/opensolaris/lib/libdtrace/common/dt_link.c

```diff
@@ -23,6 +23,7 @@
  * Copyright 2008 Sun Microsystems, Inc.  All rights reserved.
  * Use is subject to license terms.
  * Copyright 2017-2018 Mark Johnston <markj@FreeBSD.org>
+ * 2019 <klaus@unixcomp.de> 
```

Comment:
The word "Copyright" should be included before the year. Also, this should include your full name (email address is optional, but highly desirable).

Reply:
thanks for hint, actually didn't want to touch the copyright-section because most code is from the Oracle- sources and should then be something like : 
 /*
 * ....
 * Copyright (c) 2008, 2018, Oracle and/or its affiliates. All rights reserved.
 * Licensed under the Universal Permissive License v 1.0 as shown at
 * http://oss.oracle.com/licenses/upl.
 */
don't know if that will be relevant for the PR ?..

Reply:
It's OK to omit it as well. The length of the contribution is a little on the light side to include one.